packages/website/.storybook/config.js

```diff
@@ -1,4 +1,9 @@
-import {configure} from '@storybook/react';
+import {configure, addParameters, setAddon} from '@storybook/react';
+import JSXAddon, {jsxDecorator} from 'storybook-addon-jsx';
+import {SimpleChartWrapper, jsxOptions} from '../storybook/storybook-utils';
+
+setAddon(JSXAddon);
+addParameters({jsx: jsxOptions});
```

Comment:
Nice. This is what removes the need for jsxOptiions?

Reply:
Ya.  There is a bug in `addWithJSX` (referenced above) that doesn't take the 3rd argument anymore.  There is an open PR to fix it from November, however it hasn't been touched.

Since all of the stories are using the exact same options, I figured that setting it globally was easier.  This way we can't forget to use it.